docs/advanced_documentation/build-guide.md

```diff
@@ -374,9 +374,9 @@ The IDE should be able to automatically detect the Visual Studio cmake configura
   Prompt`).
 ```
 
-## Example Setup for macOS (Big Sur)
```

Comment:
we sent out a notice when upgrading macos that we officially don't support Big Sur anymore